### PR TITLE
[DO NOT MERGE] Update UnitTests.proj

### PR DESCRIPTION
**Pull request by dkurepa**
Testing

### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/core-eng/tree/master/Documentation/Validation

### CODE REVIEW

File: tests/UnitTests.proj [hunk 1]
```diff
@@ -58,10 +59,6 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(HelixAccessToken)' != '' ">
```
Comment (AraHaan) [inline tests/UnitTests.proj:61]: Why is this ItemGroup still here when everything looks to be removed from it?

File: tests/UnitTests.proj [hunk 2]
```diff
@@ -71,10 +68,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition=" '$(HelixAccessToken)' == '' ">
-    <HelixTargetQueue Include="Debian.9.Amd64.Open"/>
-    <HelixTargetQueue Include="RedHat.7.Amd64.Open"/>
-    <HelixTargetQueue Include="Windows.10.Amd64.Open"/>
-    <HelixTargetQueue Include="(Debian.10.Amd64.Open)ubuntu.1804.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-10-helix-amd64-20210304164434-56c6673"/>
```
Comment (AraHaan) [inline tests/UnitTests.proj:77]: Why was these removed?

File: tests/UnitTests.proj (hunk 1 above)
Comment (missymessa) [reply tests/UnitTests.proj:61]: It looks like @dkurepa is attempting to test some changes on our staging environment and wants to isolate it to particular queues. Given that he put "do not merge" in the PR title, I don't think he's planning to merge these changes into the repo.

**Comment by premun** [comment]
@dkurepa I guess this one can be closed?
